Dockerfile

```
@@ -41,8 +69,10 @@ RUN mv /etc/postfix/master.cf /etc/postfix/master.cf.orig
 COPY ./src/templates ./templates/
 COPY ./src/docker-entrypoint.sh ./src/version.txt ./
```

Possibly out of scope but a consistency thing with other Docker projects:

```suggestion
COPY src/templates templates/
COPY src/docker-entrypoint.sh src/version.txt ./
```